Templates/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj

```diff
@@ -2,18 +2,15 @@
   <PropertyGroup>
     <BaseOutputPath>..\..\Artifacts\MonoGame.Templates.CSharp</BaseOutputPath>
     <PackageType>Template</PackageType>
-
     <Title>MonoGame project templates</Title>
-    <Description>A set of C# templates written for the MonoGame framework.</Description>
+    <Description>This package contains a set of C# templates written for the MonoGame framework.</Description>
```

Comment:
I prefer how this was before.

Reply:
Do we need to specify it is for Visual Studio?  Or are these templates usable for other IDEs like VSCode or Ryder ?

Reply:
They're not tied to any IDE. You run them from the command line. That's true for all template packages on nuget so I don't think we need to specify that.